app/src/main/java/org/mozilla/focus/tips/TipManager.kt

```diff
@@ -33,6 +34,13 @@ class Tip(val id: Int, val text: String, val shouldDisplay: () -> Boolean, val d
         private const val FORCE_SHOW_DISABLE_TIPS_LAUNCH_COUNT = 2
         private const val FORCE_SHOW_DISABLE_TIPS_INTERVAL = 30
 
+        fun createAllowlistTip(context: Context): Tip {
+            val id = tip_explain_allowlist
+            val name = context.resources.getString(id)
+
+            return Tip(id, name, { true })
```

Comment:
We'll have to remember to figure this out once we build out the allowlist feature

Reply:
#3833 so we don't forget